packages/iris-grid/src/sidebar/conditional-formatting/ConditionalFormattingUtils.ts

```diff
@@ -653,10 +653,10 @@ export function getFormatColumns(
   rules: FormattingRule[]
 ): CustomColumn[] {
   const result: CustomColumn[] = [];
-  const ruleMap = {
-    [FormatterType.ROWS]: new Map<string, [string, CustomColumn]>(),
-    [FormatterType.CONDITIONAL]: new Map<string, [string, CustomColumn]>(),
-  };
+  // There can be only one row format custom column
+  // and multiple column format custom columns (one per column)
+  let rowFormatConfig = undefined as [string, CustomColumn] | undefined;
```

Comment:
Don't need to include all these `undefined` parts in there.
```suggestion
  let rowFormatConfig: [string, CustomColumn];
```